### PR TITLE
Show to users only avalible commands

### PR DESCRIPTION
**Pull request by desa85**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,8 @@ sudo: false
 notifications:
   email: false
 node_js:
-  - "8" # latest stable
-  - "6" # LTS
-  - "4" # maintenance
+  - "10" # latest stable
+  - "8" # maintenance
```
Comment (polina-popova) [inline .travis.yml]: Why are this changes here? Are they needed for changing command output?

File: package.json [hunk 2]
```diff
@@ -21,6 +21,10 @@
   "bugs": {
     "url": "https://github.com/hubotio/hubot-help/issues"
   },
+  "dependencies": {
+    "moment": "^2.22.2",
+    "hubot-routines": "https://github.com/tolstoyevsky/hubot-routines.git"
+  },
```
Comment (polina-popova) [inline package.json:27]: There is the same question as upper.

**Review by polina-popova — COMMENTED** [review]
Let's cut out the `privileged: admins only` from helper output - it is no need in this any more.
And, please, provide more clear description to your commit and PR.

File: .travis.yml (hunk 1 above)
Comment (desa85) [reply .travis.yml]: Here Travis settings are transferred to the requirements of our repository.

File: package.json (hunk 2 above)
Comment (desa85) [reply package.json:27]: Yes, there was a need for a separate module to check the administrator

**Review by polina-popova — COMMENTED** [review]
Please, clean code from debug printing.

File: src/help.js [hunk 3]
```diff
@@ -120,17 +121,17 @@ var stringFormatting = function stringFormatting (str) {
  * @returns {object}
  */
 var groupByMarkerGroupName = function groupByMarkerGroupName (robot, commands, markerName) {
```
Comment (polina-popova) [inline src/help.js]: I guess this alphabetic order should appear in the same pull request as function itself.

File: src/help.js [hunk 4]
```diff
@@ -173,6 +174,18 @@ var groupByMarkerGroupName = function groupByMarkerGroupName (robot, commands, m
   return commandsObject
 }
 
+var filterStatus = async function filterStatus (robot, groups, userName) {
```
Comment (polina-popova) [inline src/help.js:186]: Lets write docstring here.

**Review by polina-popova — DISMISSED** [review]

**Review by eugulixes — DISMISSED** [review]

**Review by eugulixes — APPROVED** [review]
